tests/EmbeddedMessageTest.php

```diff
@@ -134,11 +134,6 @@ public function testEmbeddedMessageWithoutContentDisposition()
         $attachment = \next($attachments);
         static::assertTrue($attachment->isEmbeddedMessage());
 
-        $embeddedMessage = $attachment->getEmbeddedMessage();
-        static::assertSame('embedded_message_subject', $embeddedMessage->getSubject());
-        static::assertNotEmpty($embeddedMessage->getBodyText());
-        static::assertNotEmpty($embeddedMessage->getBodyText());
```

Comment:
I'm sorry I've not cleared it enough: this part is very useful.
I intended to delete only the line 140 which is duplicated.
Please restore the other 3 rows :+1:

Reply:
I was surprised why u want to remove this three lines... but you have 'contrib' label ;) --- restored